Maddy-maddigordon757/Homework-1/fizzbuzz/app.js

```diff
@@ -9,30 +9,33 @@
 
 // ##### Step 1:
 // Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
-
-for (var i=1; i<100; i++)
-{
-    console.log(i);
+for (var i = 1; i <= 100; i++) {
+    if (i % 3 === 0 && i % 5 === 0) {
+        console.log("fizzbuzz")
+        }    
+        else {
+    if (i % 3 === 0) {
+        console.log("fizz")
+    } 
+    else {
+        if (i % 5 === 0) {
+            console.log("buzz")
+    }
+    else {
+        console.log(i)
+    }
```

Comment:
Consistent indents can help us ensure that we have closed all of our blocks at the end of the code. I encourage you to add the same number of spaces each time you open a new block, and go back out that number of spaces when the block closes. Then the end of your code should be something like this:
```
      }
    }
  }
}
```